api/krusty/legacy_order_test.go

```diff
@@ -0,0 +1,60 @@
+package krusty_test
+
+import (
+	"testing"
+
+	kusttest_test "sigs.k8s.io/kustomize/api/testutils/kusttest"
+)
+
+func TestIssue4388(t *testing.T) {
+	th := kusttest_test.MakeHarness(t)
```

Comment:
The test harness doesn't seem to run with legacy sort order by default, so this is not a proper regression test. Working on it.